tools/c7n_azure/tests/test_azure_events.py

```diff
@@ -51,3 +53,15 @@ def test_get_event_operations_both_event_types(self):
         self.assertEqual(len(event_operations), 2)
         self.assertTrue('Microsoft.Compute/virtualMachines/write' in event_operations)
         self.assertTrue('Microsoft.Web/serverFarms/write' in event_operations)
+
+    @arm_template('storage.json')
+    def test_create_azure_event_subscription(self):
+        account = self.setup_account()
+        queue_name = 'cctestevensub'
+        StorageUtilities.create_queue_from_storage_account(account, queue_name)
```

Comment:
nit: you can create queue as a part of storage.json provisioning and use it (no need to test queue creating twice)

Reply:
This is testing the the azure_event.py files. I am testing an event subscription gets created and there is no guarantee one test gets called before the other. There is also no way to provision a storage queue from an ARM template, so this is the only way to set it up. Notice I am not asserting the queue was created here, only asserting the event subscription. The other test is responsible for ensuring the queue gets created.